Packs/CyberArkPAS/Integrations/CyberArkPAS/CyberArkPAS.py

```diff
@@ -0,0 +1,281 @@
+                    automatic_management_enabled='true', manual_management_reason='', remote_machines='',
+                    access_restricted_to_remote_machines='false'):
+        demisto.debug('Adding a new Account')
+
+        headers = {
+            'Authorization': demisto.getIntegrationContext()['token']
+        }
+
+        body = {
+            "userName": user_name,
+            "address": address,
+            "platformId": platform_id,
+            "safeName": safe_name
+        }
+
+        if name:
+            body['name'] = name
+
+        if secret and secret_type:
+            body['secret'] = secret
+            body['secretType'] = secret_type
+
+        if platform_account_properties:
+            body['platformAccountProperties'] = platform_account_properties
+
+        if automatic_management_enabled == "false":
+            body['secretManagement'] = {
+                "automaticManagementEnabled": automatic_management_enabled,
+                "manualManagementReason": manual_management_reason
+            }
+
+        if remote_machines:
+            body['remoteMachinesAccess'] = {
+                "remoteMachines": remote_machines,
+                "accessRestrictedToRemoteMachines": access_restricted_to_remote_machines
+            }
+        res = self._http_request(
+            "POST",
+            url_suffix='/PasswordVault/api/Accounts',
+            resp_type="json",
+            headers=headers,
+            data=body
+        )
+        return res
+
+    def delete_account(self, account_id):
+        demisto.debug('Deleting an Account')
+        headers = {
+            'Authorization': demisto.getIntegrationContext()['token']
+        }
+        res = self._http_request(
+            "DELETE",
+            url_suffix='/PasswordVault/api/Accounts/' + account_id,
+            resp_type="content",
+            headers=headers,
+        )
+        return res
+
+
+'''' Commands '''
+
+
+def test_module(client):
+    client.get_system_summary()
+    return 'ok'
+
+
+def list_accounts(client, args):
+    raws = []
+    cyberark_ec = []
+    raw_response = client.get_accounts(offset=args.get('offset', '0'), limit=args.get('limit', '50')).get('value')
+
+    if raw_response:
+        for item in raw_response:
+            raws.append(item)
+            cyberark_ec.append({
+                'AccountName': item.get('name'),
+                'UserName': item.get('userName'),
+                'PlatformID': item.get('platformId'),
+                'SafeName': item.get('safeName'),
+                'AccountID': item.get('id'),
+                'CreatedTime': item.get('createdTime')
+            })
+
+    if not raws:
+        return_outputs(f'{INTEGRATION_NAME} - Could not find any Accounts', {}, {})
+
+    return CommandResults(
+        outputs_prefix='CyberArk.Accounts',
+        outputs_key_field='AccountID',
+        outputs=cyberark_ec,
```

Comment:
```suggestion
        outputs=cyberark_ec,
        raw_response=raw_response,
        readable_output=tableToMarkdown("CyberArk PAS accounts",  cyberark_ec)
```

Please add the raw_response and readble_output to the CommandResults - these create the commands raw response section as well as the markdown table printed int the war room.

Same for other uses of CommandResults please.